azure/index.md

```diff
@@ -32,10 +32,10 @@ layout: default
 
 1.  Create a resource Group with a name `kadenaResourceGroup`.
 2.  Create a Storage Account with the name `kadenaStorage`.
-3.  Spin up a VM with Kadena's ScalableBFT Image with the following requirements. (See [VM Requirements](#vm-requirements)). This will serve as the Ansible monitor VM. <https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal?toc=%2fazure%2fvirtual-machines%2flinux%2ftoc.json>
```

Comment:
Deleted link to Azure quick start because I wasn't sure how it was relevant. We can put it back though